pyintegration/deephaven2/pandas.py

```diff
@@ -0,0 +1,121 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" This module supports the conversion between Deephaven tables and Pandas DataFrames. """
+import re
+from typing import List
+
+import jpy
+import numpy as np
+import pandas
+
+from deephaven2 import DHError, new_table, dtypes
+from deephaven2.column import Column
+from deephaven2.numpy import _column_to_numpy_array, freeze_table, _make_input_column
+from deephaven2.table import Table
+
+_JPrimitiveArrayConversionUtility = jpy.get_type("io.deephaven.integrations.common.PrimitiveArrayConversionUtility")
+
+
+def _column_to_series(table: Table, col_def: Column) -> pandas.Series:
+    """ Produce a copy of the specified column as a pandas.Series object.
+
+    Args:
+        table (Table): the table
+        col_def (Column):  the column definition
+
+    Returns:
+        pandas.Series
+
+    Raises:
+        DHError
+    """
+    try:
+        data_col = table.j_table.getColumn(col_def.name)
+        np_array = _column_to_numpy_array(col_def, data_col.getDirect())
+
+        return pandas.Series(data=np_array, copy=False)
+    except DHError:
+        raise
+    except Exception as e:
+        raise DHError(e, message="failed to create apandas Series for {col}") from e
+
+
+def to_pandas(table: Table, cols: List[str] = None) -> pandas.DataFrame:
```

Comment:
Is there a reason this isn't called `to_dataframe`?  A pandas series is specified in the naming of the above function.

Reply:
Additionally, regarding the naming convention of a few functions, do we want these names to specify the type of input objects?  e.g. `to_numpy`, `to_pandas`, `to_table`.  Should these be `table_to_numpy`, `table_to_dataframe`, `dataframe_to_table`, etc?  It may not be necessary when the module names reflect the modules/objects supported, but if a user is using more than one, there could be confusion.

@chipkent 

@chipkent

Reply:
the name to_pandas echoes to_numpy better, but we certainly can go either way